monolithe/generators/lang/javascript/writers/apiversionwriter.py

```diff
@@ -99,25 +99,33 @@ def perform(self, specifications):
     def _write_locales(self, specifications):
         if self.locale_on:
             for rest_name, specification in specifications.items():
-                enum_attrs_for_locale_template = {}
-                enum_attrs = self.enum_attrs_for_locale[specification.entity_name]
-                
-                generic_enum_attrs = self.generic_enum_attrs_for_locale[specification.entity_name];
-                if (generic_enum_attrs):
-                    enum_attrs.extend(generic_enum_attrs)
-                    
-                if (enum_attrs):
-                    for attribute in enum_attrs:
-                        enum_name = "%s%s%sEnum" % (self._class_prefix, specification.entity_name, attribute.name[0].upper() + attribute.name[1:])
-                        enum_attrs_for_locale_template[enum_name] = attribute.allowed_choices
+                #enum_attrs_for_locale_template = {}
```

Comment:
if this is not required please remove

Reply:
I will use this shortly for enum locales, hence retaining as comments for now